### DISCUSSION TITLE
How can I show the git blame in the status line, and the buffers in tabs?

### DISCUSSION THREAD

**Discussion by ghost** [famiu/feline.nvim#208] [Q&A] [answered]
I'm moving from airline to feline and I would like to know how to have the same configuration in feline as I have in airline

    Plug 'vim-airline/vim-airline' " Fancy statusline
    Plug 'vim-airline/vim-airline-themes' " Themes for vim-airline
    let g:airline_powerline_fonts=1
    let g:airline_theme='onedark'
    let g:airline_section_c = "%{get(b:,'coc_git_blame','')}"
    let g:airline#extensions#tabline#enabled = 1
    let g:airline#extensions#tabline#tab_min_count = 2
    let g:airline#extensions#tabline#show_buffers = 1
    let g:airline#extensions#tabline#show_splits = 0

**Comment by ram02z** [comment, marked as answer]
Just to preface, I have not used airline.
```vim
let g:airline_powerline_fonts=1
```

By default, providers include an icon where possible. If you wanted to change the icon, you can change the `icon` value.
See https://github.com/feline-nvim/feline.nvim/blob/master/USAGE.md#component-icon
For separators between providers, you can use one of the presets or set a custom string value. 
See the default separators, https://github.com/feline-nvim/feline.nvim/blob/master/USAGE.md#separator-presets
For separator example, see https://github.com/feline-nvim/feline.nvim/blob/master/USAGE.md#component-separators
***
```vim
let g:airline_theme='onedark'
```
You can create a new theme using the onedark colors if you aren't happy with the defaults.
See https://github.com/feline-nvim/feline.nvim/blob/master/USAGE.md#adding-new-themes
***
```vim
let g:airline_section_c = "%{get(b:,'coc_git_blame','')}"
```
You can create a custom provider like so:
```lua
{
  provider = function()
      return vim.b.coc_git_blame
  end,
  enabled = function()
    return vim.b.coc_git_blame ~= nil
  end
}
```
***
```vim
let g:airline#extensions#tabline#enabled = 1
let g:airline#extensions#tabline#tab_min_count = 2
let g:airline#extensions#tabline#show_buffers = 1
let g:airline#extensions#tabline#show_splits = 0
```
Unforunately, tabline support has not been implemented yet. See #169 

There are plugins for Neovim that allow buffers to be shown on tabline: 
https://github.com/jose-elias-alvarez/buftabline.nvim
https://github.com/akinsho/bufferline.nvim
https://github.com/romgrk/barbar.nvim
